jvb/src/main/java/org/jitsi/videobridge/Conference.java

```diff
@@ -219,6 +225,35 @@ public Conference(Videobridge videobridge,
         videobridgeStatistics.totalConferencesCreated.incrementAndGet();
         epConnectionStatusMonitor = new EndpointConnectionStatusMonitor(this, TaskPools.SCHEDULED_POOL, logger);
         epConnectionStatusMonitor.start();
+
+        colibriQueue= new PacketQueue<>(
+            100,
+            true,
+            "colibri-queue-" + id,
+            request ->
+            {
+                try
+                {
+                    // TODO: we can avoid reaching into Videobridge here by merging VideobridgeShim into ConferenceShim
+                    IQ response = videobridge.handleColibriConferenceIQ(this, request.getRequest());
+                    long delay = System.currentTimeMillis() - request.getReceiveTime();
+                    if (delay > 100)
```

Comment:
Current delay is just the processing time of the IQ itself, not the time since the request was received - i.e. doesn't include the time the request was queued. I think we still want that.

Reply:
Why do we want just the processing and not the queueing delay? I can see it can be useful when we're profiling, but the overall delay in the response seems like a more useful metric to have in production.

Reply:
Well, the existing logs and stats show just the processing delay, no?  Since any queueing happens inside the XMPP stack.